dss/util/auth/auth0.py

```diff
@@ -16,7 +16,7 @@ class FlacMixin(Authorize):
     access control (FLAC) table to check if a user is allowed
     to access a given UUID.
     """
-    ddb_table_name = f"dss-auth-lookup-${os.environ['DSS_DEPLOYMENT_STAGE']}"
+    flac_lookup_table_name = f"dss-auth-lookup-${os.environ['DSS_DEPLOYMENT_STAGE']}"
```

Comment:
Should this really be `lookup-${...}` or should it just be `lookup-{...}`?

Reply:
yes! glad you caught that, i had a PR with that change, must have gotten trimmed